src/test/java/com/softserve/edu/greencity/ui/tests/signup/RegistrationTests.java

```diff
@@ -154,15 +146,5 @@ public void existingUserRegistration(User userLoginCredentials) {
         softAssert.assertAll();
     }
 
-    @Test(dataProvider = "invalidPasswordDataProvider", testName = "GC-204")
```

Comment:
In reality it was GC-516
This method is wrong and assertions not right. It duplicates GC-517, so I merged them.